### PR TITLE
🚚 Move all TS exports to @huggingface/tasks

### PR DESCRIPTION
**Pull request by coyotte508**

### CODE REVIEW

**Review by julien-c — APPROVED** [review]

**Review by julien-c — COMMENTED** [review]
@coyotte508 if you want you can move the many folders (one for each task) into a `task-pages` subfolder

It's a bit messy to have them at the root IMO

**Review by julien-c — COMMENTED** [review]
lets try to minimize the file layout diff compared to before

**Comment by mishig25** [comment]
Nit: this [README](https://github.com/huggingface/huggingface.js/blob/refacto-end/packages/tasks/src/README.md?plain=1) is assuming that there is a folder called `Interfaces`

File: packages/tasks/src/tasks/audio-classification/data.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import type { TaskDataCustom } from "../Types";
+import type { TaskDataCustom } from "..";
```
Comment (mishig25) [inline packages/tasks/src/tasks/audio-classification/data.ts:1]: there has to be something here right ?

Comment (mishig25) [reply packages/tasks/src/tasks/audio-classification/data.ts:1]: Oh I see that `index.ts` is being inferred

**Comment by mishig25** [comment]
No string preference. But we can leave [Widgets Input/Output types](https://github.com/huggingface/huggingface.js/blob/refacto-end/packages/tasks/src/model-data.ts#L5-L25) in hf.js/widgets rather than moving them to hf.js/tasks ?

**Pull request merged**